text/0092-logs-vision.md

```diff
@@ -96,6 +96,16 @@ This is important because we believe software that was created before
 OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
 efforts where possible.
 
+Specifically, we should be able to collect the following sources:
+- Log files, especially those emitted by container runtimes
+- Syslog
+- Systemd/Journald
+- Windows Event Log
+- Event Tracing for Windows (ETW)
+- JSON over TCP or UDP
```

Comment:
We use raw JSON over TCP at AWS with some libraries: https://github.com/awslabs/aws-embedded-metrics-python

Anecdotally, having worked with Fluentd and Fluent Bit for 2 years, these inputs are rare/niche compared to others, but definitely in use by a good number of people.

Reply:
These are probably good fit for the Contrib. I do not think they should be part of the vision or core roadmap. Collector currently encourages to actively contribute components that support specific formats to Contrib repo and we already see this happening for traces and metrics. I believe we need to continue the practice for logs. If anyone is interested in these particular formats they should be able to add these as components.